source/index.html.md

```diff
@@ -102,20 +107,24 @@ The <code>X-Trace-ID</code> should also be provided, where possible, for support
 ```shell
 curl \
 -H "Content-Type: application/json" \
--H "X-API-Key: [vendor-api-key]" \
--H "X-Idempotency-Key: [vendor-generated-key]" \
+-H "X-API-Key: [partner-api-key]" \
+-H "X-Idempotency-Key: [partner-generated-key]" \
 ...
 ```
 
-`X-Idempotency-Key: [vendor-generated-key]`
+`X-Idempotency-Key: [partner-generated-key]`
 
-The API supports idempotency for safely retrying requests without accidentally performing the same operation twice. A unique identifier added to the header on `POST` or `PUT` requests ensures that the same request is not mistakenly processed multiple times. If a duplicate request is sent then the original response will be returned without any processing occurring. This includes all response codes including `500` errors, so a new key should be created for each independent request. The Betable API will store the idempotent response for 30 days.
```

Comment:
> This includes all response codes including `500` errors, so a new key should be created for each independent request.

I felt like the above sentence was confusing. Did it mean that `500` errors will be returned without processing? That doesn't sound right, kind makes using idempotency harder. We should only be storing results when a `200`/`201/`204` is returned for idempotency. If they get a `500` they can just straight retry the same request without doing any changes on their side.

Reply:
Ok.  I copies that as it was the approach Stripe did but I think since we also have X-Trace-ID that it is unnecessary to require a new idempotency key.